oak_enclave_rt/src/lib.rs

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#![no_std]
+
+use heap::LockedGrowableHeap;
+
+mod heap;
+mod libm;
+
+#[cfg(not(test))]
```

Comment:
Will external reviewers be sure that the release version doesn't contain the test code?

Reply:
It's the same code, just that we don't set it as a global allocator.